core/types/accounts/account.go

```diff
@@ -94,18 +88,9 @@ func (a *Account) EncodingLengthForHashing() uint {
 
 	structLength += uint(balanceBytes + nonceBytes + 2)
 
+	//todo why do we need it?
```

Comment:
Because in the convention for hashing, the storage root and the code hash always need to be present, and they are always fixed size (1+32+1+32)

Reply:
Thank you! I've just forgot to remove.

Reply:
You can actually turn into the comment